run.sh

```diff
@@ -45,6 +45,9 @@ function _cleanup() {
 	elif [[ $LAN_PARENT =~ \. ]] ; then
 		sudo ip link del dev $LAN_PARENT
 	fi
+	echo "* Rolling back ip address for main if"
+	sudo service dhcpcd start
+	sudo dhclient -r
```

Comment:
1. I'm not sure I understand the reason why this is necessary in your case.

2. While `dhcpcd` is usually run as a separate service on Raspberry Pi OS, this is not the case for other intended targets, so I'd rather add some checks to this first.

Also, I would prefer not to add an additional dependency on `dhclient` if we are already using `dhcpcd`. They are not the same thing.